pubtools/_content_gateway/rough.py

```diff
@@ -0,0 +1,59 @@
+# from utils import yaml_parser
```

Comment:
???

Reply:
Sorry, my bad. This file shouldn't be added.